config/build.sh

```diff
@@ -42,6 +42,35 @@ for OPERATOR_SDK_VERSION in "${!OSDK_VERSION_HASHES[@]}"; do
     mv $OPERATOR_SDK_BINARY /usr/local/bin
 done
 
+###############
+# Set up go env
+###############
+# Get rid of -mod=vendor
+unset GOFLAGS
+# No, really, we want to use modules
+export GO111MODULE=on
+
+#############
+# openapi-gen
+#############
+go get k8s.io/code-generator/cmd/openapi-gen@v0.19.4
```

Comment:
Can we define all of these various versions into variables to the top of the file? It will make maintaining this much easier in the future.

Reply:
Done.